markdown/events/2022/bytesize-nascent.md

```diff
@@ -23,3 +23,107 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[10:18](https://www.youtube.com/watch?v=chayGGPTnfM&t=618)
+The first thing is we're grouping all the replicates up. Basically that's anything that's a technical replicate that we want to group up, to increase the signal and biological replicates. Then we feed that into for GRO-seq over here, if they're specifically, because that's what I've been so interested in. We feed that into Homer and GroHMM optionally, and we'll talk a little bit more about that. Everything else that's a transcriptional start site and GRO-seq and others. We feed that into PINTS as well, and then we go into bed tools and we can intersect the two of these with a filter and without a filter, and then basically only call regions that we're interested in and drop the regions that we're not. We can drop the regions that are gene bodies and promoters because we know that those aren't going to be eRNAs or other interesting RNAs. We can also make sure that we keep only regions that we're interested in, such as like those with histone modifications that indicate eRNAs. Then we just do some quick quantification, and then we move into MultiQC.
+
+[11:29](https://www.youtube.com/watch?v=chayGGPTnfM&t=689)
+Another little added benefit that we were interested in was supporting CHM13, which is a new reference genome that came out recently. Highly recommend y'all look into that as well, if you're interested in that. I'll be adding this to the template soon. But the main thing here in this infographic that they found is they were specifically looking at methylation data and how the new reference improved calls.
+
+[11:56](https://www.youtube.com/watch?v=chayGGPTnfM&t=716)
+You can see over here on the left is the number of MACS peaks, and then you can see the blue is the old reference. The CHM13 reference are the additional calls that were made from using this reference. So these may not be much and may not be of interest in things that are well known and well understood, but very relevant for nascent transcript calling. We have support for that in our IGMs config, and you can just use CHM13 and align to that.
+
+[12:29](https://www.youtube.com/watch?v=chayGGPTnfM&t=749)
+Let's talk a little bit about the transcript identification because that's the most interesting part of the pipeline and what makes it unique. There's a couple of different options, as I said. First, if you're doing GRO-seq, I have some great support for that. If anyone would like to support other assays or would like to see it supported, please open an issue. So first is GroHMM, and this is what kind of sparked us getting into the nextflow and going into bigger pipelines. It was difficulty reproducing this and running this on big enough machines to actually use it because it's an R package. It was released in 2015 by Minho Chae, Charles Danko, and Lee Kraus, actually just down the street at UT Southwestern.
```

Comment:
```suggestion
Let's talk a little bit about the transcript identification because that's the most interesting part of the pipeline and what makes it unique. There's a couple of different options, as I said. First, if you're doing GRO-seq, I have some great support for that. If anyone would like to support other assays or would like to see it supported, please open an issue. So first is GroHMM, and this is what kind of sparked us getting into the nextflow and going into bigger pipelines. It was difficult reproducing this and running this on big enough machines to actually use it because it's an R package. It was released in 2015 by Minho Chae, Charles Danko, and Lee Kraus, actually just down the street at UT Southwestern.
```